app/gui/view/debug_scene/icons/src/lib.rs

```diff
@@ -0,0 +1,90 @@
+// === Standard Linter Configuration ===
+#![deny(non_ascii_idents)]
+#![warn(unsafe_code)]
+
+use ensogl::system::web::traits::*;
+use ide_view_component_group::prelude::*;
+use wasm_bindgen::prelude::*;
+
+use ensogl::application::Application;
+use ensogl::data::color;
+use ensogl::display::navigation::navigator::Navigator;
+use ensogl::display::DomSymbol;
+use ensogl::system::web;
+use ide_view_component_group::icon;
+use ide_view_component_group::icon::ICON_SIZE;
+use ide_view_component_group::icon::SHRINK_AMOUNT;
+
+
+
+// =============
+// === Frame ===
+// =============
+
+/// A rectangular frame to mark the edges of icon.
```

Comment:
What is this frame and where is it used? This sohuld be explained. After reading this comment I dont know what this thing refers to as I dont remember anything like that in the desing.